### DISCUSSION TITLE
Doris 1.2.0 jdbc_catalog 使用报错

### DISCUSSION THREAD

**Discussion by Qichen-Kylin** [apache/doris#19510] [A - General / Q&A]
CREATE CATALOG tidb PROPERTIES (
    "type"="jdbc",
    "user"="",
    "password"="",
    "jdbc_url" = "jdbc:mysql://:4000/testdb?useSSL=false&serverTimezone=Asia/Shanghai&allowPublicKeyRetrieval=true",
    "driver_url" = "hdfs://doris/lib/mysql-connector-java-5.1.47.jar",
    "driver_class" = "com.mysql.jdbc.Driver"
);

mysql> SWITCH tidb;
Query OK, 0 rows affected (0.00 sec)

mysql> show databases;
ERROR 1105 (HY000): JdbcClientException, msg: org.apache.doris.external.jdbc.JdbcClientException: Unsupported jdbc database type, please check jdbcUrl: ?yearIsDateType=false
这里就开始报错，需检查 jdbcUrl

**Comment by LemonLiTree** [comment]
Please upgrade to 1.2.4 for testing

**Comment by zy-kkk** [comment]
首先，你的jdbc url是错误的，
请参考:https://doris.apache.org/zh-CN/docs/1.2/lakehouse/multi-catalog/jdbc#%E5%88%9B%E5%BB%BA-catalog
其次，1.2.0的 jdbc catalog功能并不完善，建议升级1.2.4.1:https://doris.apache.org/zh-CN/download/
